hudi-flink/src/test/java/org/apache/hudi/table/TestHoodieTableFactory.java

```diff
@@ -297,4 +343,4 @@ public boolean isTemporary() {
       return false;
     }
   }
-}
+}
```

Comment:
nit: new line

Reply:
the new line check the last line of the file, should be like this
```
}

```

Reply:
done